test/extended/image_ecosystem/mysql_replica.go

```diff
@@ -184,7 +196,7 @@ func replicationTestFactory(oc *exutil.CLI, tc testCase) func() {
 			g.By("after slave is scaled to 0 and then back to 4 replicas")
 			err = oc.Run("scale").Args("dc", "mysql-slave", "--replicas=0").Execute()
 			o.Expect(err).NotTo(o.HaveOccurred())
-			err = exutil.WaitUntilPodIsGone(oc.KubeClient().CoreV1().Pods(oc.Namespace()), pods.Items[0].Name, 1*time.Minute)
+			err = exutil.WaitUntilPodIsGone(oc.KubeClient().CoreV1().Pods(oc.Namespace()), pods.Items[0].Name, 2*time.Minute)
```

Comment:
yuck

Reply:
from what i can tell this is necessary since we started running these tests in parallel and overloading the cluster.

however i also think general pod scheduling/container start times have gotten worse independent of that, but i can't fight that battle right now.

so yeah, agreed on yuck.

Reply:
also yuck on commented out test code.... just delete and `git revert` later. no need for stale commented code

Reply:
a lot of people make that claim about commented out code(just delete it, git will hold into it for you), but how many people, when looking at a chunk of code, think "let me go check the git history to see if anything useful was deleted in a previous commit that i should take a look at and potentially revive/reuse it here?"  I don't get it.  Commented out code represents "this is of interest to someone working in this area and i want to be sure they see it when they are touching this code before they go do something else/redo this on their own"

(and this PR isn't commenting out the code, it was already commented out).

Reply:
Asking someone making future changes to this test to refactor code that does not run and cannot be tested is not very nice of whoever is commenting it out in the first place.

Reply:
it's a starting point to consider, not a solution on a silver platter.